app/src/main/java/com/duckduckgo/app/global/api/AtbAndAppVersionPixelRemovalInterceptor.kt

```diff
@@ -32,7 +33,14 @@ import okhttp3.Response
     scope = AppScope::class,
     boundType = PixelInterceptorPlugin::class,
 )
-class AtbAndAppVersionPixelRemovalInterceptor @Inject constructor() : Interceptor, PixelInterceptorPlugin {
+class AtbAndAppVersionPixelRemovalInterceptor @Inject constructor(
+    private val pixelsPlugin: PluginPoint<PixelRequiringDataCleaningPlugin>,
+) : Interceptor, PixelInterceptorPlugin {
+
+    val pixels: List<String> by lazy {
+        pixelsPlugin.getPlugins().flatMap { it.names() }.distinct()
```

Comment:
nit: `distinct()` converts to a set and then back to a list. The second (to list) conversion is not really necessary. We can just `toSet()` instead